src/ruby/.rubocop_todo.yml

```diff
@@ -368,6 +368,7 @@ Style/GlobalVars:
 # Configuration parameters: MinBodyLength.
 Style/GuardClause:
   Exclude:
+    - 'end2end/package_with_underscore_checker.rb'
```

Comment:
are the rubocop error in this file pre-existing? if so, this is suprising by excluding would be ok.

If they aren't, can you please fix them in this PR instead?

Reply:
I fixed it https://github.com/grpc/grpc/pull/15377/commits/c2e5fbee44beabfcdcee942a644c5da11b13f8c5